sklearn/titanic/README.md

```diff
@@ -0,0 +1,125 @@
+gcloud auth application-default login
+```
+
+##### 3. Running the Code
+We need to make the dataset available to the training code. Here are two ways for the code to access the dataset.
+
+###### Option 1: Copying it to the instance
+We can copy the dataset into the new instance and use it directly inside the instance and read it as a local file. Fortunately, *gcloud* makes this an easy step:
+```bash
+gcloud compute scp ./train.csv $MY_INSTANCE:~/
+```
+
+Note that the same method can be used to copy any other file (e.g. the python code):
+```bash
+gcloud compute scp ./titanic.py $MY_INSTANCE:~/
+```
+
+
+We can then run our python code in the instance to create a model:
+```bash
+# Run on the new instance
+python titanic.py --titanic-data-path ~/train.csv --model-output-path ~/model.pkl
+```
+
+which will generate the model and save it on the instance.
+
+###### Option 2: Reading it from GCS
+Another solution is to read the dataset directly from a bucket in GCS. We use *gsutil* to create the bucket in GCS, and then copy the dataset into it:
+```bash
+gsutil mb $MY_BUCKET
+gsutil cp ./train.csv  gs://$MY_BUCKET/$MY_FOLDER/train.csv
+```
+The advantage of using *gfile* in our python code is that we can use the exact same code and read the dataset from GCS:
+```bash
+# Run on the new instance
+export $MY_BUCKET=THE_GIVEN_BUCKET_NAME_HERE
+export $MY_FOLDER=THE_GIVEN_FOLDER_NAME_HERE
+python titanic.py --titanic-data-path gs://$MY_BUCKET/$MY_FOLDER/train.csv --model-output-path gs://$MY_BUCKET/$MY_FOLDER/model.pkl
+```
+
+which will create the model and store it in the same location as our dataset in *GCS*.
+
+That is it!! We have successfully trained a classifier using scikit-learn on a *GCE* instance.
+
+##### 4. Stopping or Deleting the Instance
+The *GCE* instance that we created will be running indefinitely and costing us, unless we either stop or delete it completely. Stopping the instance will allow us to start it again later on, should we require to use it again which may be the better choice:
+```bash
+gcloud compute instances stop $MY_INSTANCE
+```
+
+However, if we want to completely delete the instance, we can use the following command:
+```bash
+gcloud compute instances delete $MY_INSTANCE
+```
+
+## What is Next
+
+##### Automating the Entire Process
+It is possible to automate the entire process described in this tutorial by writing some scripts to be run periodically. The [burst training sample](https://github.com/GoogleCloudPlatform/ml-on-gcp/tree/master/gce/burst-training) has a detailed description of how to do this.
+
+##### Lowering the Cost
+You may sometimes require an instance with a high number of CPU's or a large amount of RAM. These instances are typically more expensive. [Preemptible Instances](https://cloud.google.com/compute/docs/instances/preemptible) can be a great option to train models on at a much lower cost, and they are definitely worth considering if you are going to train certain models periodically.
```

Comment:
Can you add a little more explanation on why Preemptible Instances will help lower cost? 
Your title is lowering cost, but then you say to use more expensive instances. 

"Preemptible Instances are a great option to train models on at a much lower cost, because once the training is complete they can be stopped right away, check out this guide. They are definitely worth considering if you are ..."
Burst Training: https://github.com/GoogleCloudPlatform/ml-on-gcp/blob/master/gce/burst-training/README.md

Reply:
Changed the text a little.